packages/docusaurus/lib/server/load/index.js

```diff
@@ -52,18 +50,53 @@ module.exports = async function load(siteDir, cliOptions = {}) {
     context,
   });
 
-  // Resolve outDir.
   const outDir = path.resolve(siteDir, 'build');
+  const {baseUrl} = siteConfig;
 
-  // Resolve theme.
+  // Resolve theme. TBD (Experimental)
   const themePath = loadTheme(siteDir);
 
-  const {baseUrl} = siteConfig;
+  // Routing
+  const {
+    routesAsyncModules,
+    routesConfig,
+    routesMetadata,
+    routesMetadataPath,
+    routesPaths,
+  } = await loadRoutes(pluginsRouteConfigs);
+
+  // Mapping of routePath -> metadataPath. Example: '/blog' -> '@generated/metadata/blog-c06.json'
+  // Very useful to know which json metadata file is related to certain route
+  await generate(
+    generatedFilesDir,
+    'routesMetadataPath.json',
```

Comment:
Are these imported anywhere or are these just generated to let us (developers) know what's going on?

Reply:
Not imported anywhere yet (so it doesnt affect main.js bundle size now)

This is because we dont include crypto package in client side. (crypto is only in nodejs). For future use if we want to know where is the hashed path